src/ng/compile.js

```diff
@@ -1387,7 +1387,11 @@ function $CompileProvider($provide, $$sanitizeUriProvider) {
    * binding information and a reference to the current scope on to DOM elements.
    * If enabled, the compiler will add the following to DOM elements that have been bound to the scope
    * * `ng-binding` CSS class
+   * * `ng-scope` and `ng-isolated-scope` CSS classes
    * * `$binding` data property containing an array of the binding expressions
+   * * `scope()` and `isolateScope()` allowing access to scope from DOM Element
```

Comment:
It doesn't add the methods. The methods are always present. It adds the `$scope` or `$isolateScope` (or `$isolateScopeNoTemplate` data properties (that can be used by the `.scope()` and `.isolateScope()` methods).